src/osd/OSDMap.cc

```diff
@@ -5331,6 +5393,113 @@ OSDMap::candidates_t OSDMap::build_candidates(
   return candidates;
 }
 
+float OSDMap::calc_wl_balance_score(CephContext *cct, int64_t pool_id,
```

Comment:
We should make it clear that this function returns the acting score, but updates all other scores as well. (Maybe in a comment above the function definition in OSDMap.h would be good)